OMCompiler/Compiler/Util/Flags.mo

```diff
@@ -1152,7 +1152,7 @@ constant ConfigFlag INST_CACHE_SIZE = CONFIG_FLAG(90, "instCacheSize",
   NONE(), EXTERNAL(), INT_FLAG(25343), NONE(),
   Gettext.gettext("Sets the size of the internal hash table used for instantiation caching."));
 constant ConfigFlag MAX_SIZE_LINEAR_TEARING = CONFIG_FLAG(91, "maxSizeLinearTearing",
-  NONE(), EXTERNAL(), INT_FLAG(200), NONE(),
+  NONE(), EXTERNAL(), INT_FLAG(0), NONE(),
   Gettext.gettext("Sets the maximum system size for tearing of linear systems (default 200)."));
```

Comment:
```suggestion
  Gettext.gettext("Sets the maximum system size for tearing of linear systems (default 0, no tearing)."));
```

Reply:
Thanks for the catch. I totally forgot to change the description.